test/realm_test.dart

```diff
@@ -486,6 +485,50 @@ Future<void> main([List<String>? args]) async {
       expect(cars[0].make, car.make);
     });
 
+    test('Query results', () {
+      var config = Configuration([Car.schema]);
+      var realm = Realm(config);
+      realm.write(() => realm
+        ..add(Car()..make = "Audi")
+        ..add(Car()));
+      final cars = realm.all<Car>().query('make == "Tesla"');
+      expect(cars.length, 1);
+      expect(cars[0].make, "Tesla");
+    });
+
+    test('Query realm', () {
```

Comment:
suggestion: rename test to "Query class"

Reply:
Fixed